packages/react-form/src/types.ts

```diff
@@ -97,6 +100,10 @@ export interface FieldBag {
   [key: string]: FieldOutput<any>;
 }
 
+export interface ListBag {
+  dynamicList: DynamicList<any>;
```

Comment:
What if I want to handle several dynamic lists? I think we need to find inspiration on how `FieldBag` works to allows that.

Reply:
I am wondering if this is something we should do in this PR? 🤔 we might have to add some additional logic like resetting all the dynamic lists inside the bag basically we might need some type of reducer like FieldBag.

Reply:
We can discuss about it but I don't think it's okay to add an incomplete feature to a public package :/

Reply:
Just added this [here](https://github.com/Shopify/quilt/pull/1828/commits/b4ab1302741510cb30e08764745d54f2be1a92e9)